syntax = "proto3"; 
package Login_Protobuf;

// message SearchResponse {
//   message Result {
//     string url = 1;
//     string title = 2;
//     int64  telephone = 3;
//     repeated string snippets = 4;
//   }
//   repeated Result results = 1;
// }

// Login_Server
message Account{
  string account = 1;
  string password = 2; 
}
message Login_Request{
  Account Logininfo = 1;
}
message Signup_Request{
  Account Signupinfo = 1;
  string Name = 2;
}

message Login_Response{
  int32 result=1;
  string name=2;
}
message Signup_Response{
  int32 result=1;
}